src/client/search/SearchTermsReducer.spec.js

```diff
@@ -0,0 +1,63 @@
+import { should } from 'chai';
+import deepFreeze from 'deep-freeze';
+import SearchTermsReducer from './SearchTermsReducer';
+
+describe(SearchTermsReducer, () => {
+  it('should add a hashtag search term', () => {
+    const stateBefore = [];
+    const action = {
+      type: 'ADD_SEARCH_TERM',
+      id: 0,
+      query: 'Football',
+      paramTypes: ['hashtag'],
+      source: 'twitter',
+    };
+
+    const stateAfter = [{
+      id: 0,
+      query: 'Football',
+      paramTypes: ['hashtag'],
+      source: 'twitter',
+    },];
+
+    deepFreeze(stateBefore);
```

Comment:
I get what this is doing, but is it totally necessary for the test?
Although tbf in another language the objects would be immutable before hand anyway